src/main/java/driver/Driver.java

```diff
@@ -108,6 +121,54 @@ public boolean exist(Locator locator) {
         }
     }
 
+    public boolean dispayed(Locator locator) {
+        exist(locator);
+        try {
+            getNativeDriver().manage().timeouts().implicitlyWait(Duration.ofSeconds(1));
```

Comment:
u`ve already used exist method, so u dont need this. check if displayed after checking if exist